### ISSUE TITLE
jsx-closing-bracket-location when returning directly

### ISSUE THREAD

**Issue by FredyC** [yannickcr/eslint-plugin-react#243]
With default settings, this is considered warning. I am wondering how do you write this kind of code.

        render() {
            return (<Application
                {...defaultProps}
                {...this.props}
                {...options}
            />);
        }

This looks totally weird in my opinion...

        render() {
            return (
            <Application
                {...defaultProps}
                {...this.props}
                {...options}
            />);
        }

**Comment by yannickcr** [comment]
I personally write it this way:
```js
    render() {
        return (
            <Application
                {...defaultProps}
                {...this.props}
                {...options}
            />
        );
    }

**Comment by FredyC** [comment]
Ah I see, that doesn't look so bad except it's two extra lines. Hmm. Do you think it would be possible to extend the rule so it can recognize alignment to return ?

**Comment by yannickcr** [comment]
It is possible to add another location type, with the same behavior of `tag-aligned` except when the tag is preceded with return.

```jsx
render() {
  return (<App
    <Component
      {...defaultProps}
      {...this.props}
      {...options}
    />
  </App>);
}
```

**Comment by csilvers** [comment]
I think it's not just `return`.  (We are seeing this same issue, and would prefer not to have the extra parens.)  For instance, we'd like to be able to do
```
   var x = <App
      ...
   />;
